[2021-08-12 10:04:47,510] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: trigger_rule.task_3 2021-08-11T00:00:00+00:00 [queued]>
[2021-08-12 10:04:47,516] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: trigger_rule.task_3 2021-08-11T00:00:00+00:00 [queued]>
[2021-08-12 10:04:47,516] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-08-12 10:04:47,516] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2021-08-12 10:04:47,516] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-08-12 10:04:47,524] {taskinstance.py:1107} INFO - Executing <Task(BashOperator): task_3> on 2021-08-11T00:00:00+00:00
[2021-08-12 10:04:47,526] {standard_task_runner.py:52} INFO - Started process 89215 to run task
[2021-08-12 10:04:47,531] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'trigger_rule', 'task_3', '2021-08-11T00:00:00+00:00', '--job-id', '26', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/trigger_dag.py', '--cfg-path', '/tmp/tmpulymvova', '--error-file', '/tmp/tmpho26h7fq']
[2021-08-12 10:04:47,532] {standard_task_runner.py:77} INFO - Job 26: Subtask task_3
[2021-08-12 10:04:47,560] {logging_mixin.py:104} INFO - Running <TaskInstance: trigger_rule.task_3 2021-08-11T00:00:00+00:00 [running]> on host adam
[2021-08-12 10:04:47,598] {taskinstance.py:1300} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=trigger_rule
AIRFLOW_CTX_TASK_ID=task_3
AIRFLOW_CTX_EXECUTION_DATE=2021-08-11T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-08-11T00:00:00+00:00
[2021-08-12 10:04:47,599] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-08-12 10:04:47,599] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'exit 0']
[2021-08-12 10:04:47,605] {subprocess.py:74} INFO - Output:
[2021-08-12 10:04:47,606] {subprocess.py:82} INFO - Command exited with return code 0
[2021-08-12 10:04:47,612] {taskinstance.py:1204} INFO - Marking task as SUCCESS. dag_id=trigger_rule, task_id=task_3, execution_date=20210811T000000, start_date=20210812T080447, end_date=20210812T080447
[2021-08-12 10:04:47,626] {taskinstance.py:1265} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-08-12 10:04:47,668] {local_task_job.py:149} INFO - Task exited with return code 0
